#	This program is to show the exception handling in python

try:
	
	a = 10 / 0
	
except:
	
	print 'Divide by zero error !'	
